makefile

```diff
@@ -387,6 +385,15 @@ scramble:
 	python cmd/scrambled-eggs/scramble.py $(file) $(multiarg)
 
 # mongodb utility targets
+mongotoolsDir := mongotools
+run-mongotools = $(if $(wildcard $(mongotoolsDir)), ./$(mongotoolsDir)/$(1), $(1))
+$(buildDir)/.get-mongotools:
+ifeq (,$(shell which mongorestore))
```

Comment:
Is this meant to be used by regular devs, or is it only for the CI testing?

If we're going to have regular devs use it to get mongo-tools, instead of doing conditional `if` logic in Make, it's more normal to just make `.get-mongotools` have a dependency on the mongotools directory existing. Otherwise, if this is only used for CI testing, I would just follow the same model as `get-mongodb`/`get-mongosh` for consistency of our CI tests.

Reply:
Done.